thermo_nerf/scripts/renderimagemodality_type.py

```diff
@@ -0,0 +1,12 @@
+from enum import Enum
+
+
+class RenderedImageModality(Enum):
+    RGB = "rgb"
+    DEPTH = "depth"
+    ACCUMULATION = "accumulation"
+    THERMAL = "thermal"
+    THERMAL_COMBINED = "thermal_combined"
+
+    def __str__(self):
```

Comment:
Don't add this function it's not needed. Use `RIM.value` in the code when needed